spec/paths/enterprise@{enterprise_id}@content@.yaml

```diff
@@ -0,0 +1,90 @@
+get:
+  operationId: getAllContent
+  summary: List content in the enterprise
+  description: Returns Content list
+  produces:
+    - application/json
+  parameters:
+    - name: tags
+      in: query
+      description: filter by tag name
+      required: false
+      type: string
+    - name: description
+      in: query
+      description: filter by description name
+      required: false
+      type: string
+    - name: limit
+      in: query
+      description: Number of results to return per page.
+      required: false
+      type: integer
+      default: 20
+    - name: offset
+      in: query
+      description: The initial index from which to return the results.
+      required: false
+      type: integer
+      default: 0
+  responses:
+    '200':
+      description: successful operation
+      schema:
+        required:
+          - count
+          - results
+        type: object
+        properties:
+          count:
+            type: integer
+          next:
+            type: string
+            format: url
+            x-nullable: true
+          previous:
+            type: string
+            format: url
+            x-nullable: true
+          results:
+            type: array
+            items:
+              $ref: '#/definitions/Content'
+  security:
+    - apiKey: []
+  tags:
+    - Content
+post:
```

Comment:
post is only allowed for 
`<content_api>/content/upload/` and content type is not application/json rather multipart/form-data
Please check application upload api `<application_api>/application/upload/` Both the same only key is different